src/tests/slot-composer-test.ts

```diff
@@ -102,27 +102,22 @@ recipe
         .expectRenderSlot('BB', 'mySlot')
         .expectRenderSlot('C', 'otherSlot')
         ;
-
     await arc.instantiate(plan);
-    await arc.idle;
     await observer.expectationsCompleted();
   });
 
-  it('initialize recipe and render hosted slots', async () => {
+  it('initialize recipe and render hosted slots', async function() {
```

Comment:
is this change intentional? i think it causes a lint error.

Reply:
Darn it. If you need `this.timeout()` you have to use a plain `function()`, but otherwise lint forces `() =>` format. I've had to keep switching in order to debug. :(